src/aabb.rs

```diff
@@ -39,41 +39,33 @@ impl<S> MinMax for Point2<S>
     where S: BaseNum
 {
     fn min(a: Point2<S>, b: Point2<S>) -> Point2<S> {
-        Point2::new(
-            a.x.partial_min(b.x),
-            a.y.partial_min(b.y)
-        )
+        Point2::new(a.x.partial_min(b.x), a.y.partial_min(b.y))
     }
 
     fn max(a: Point2<S>, b: Point2<S>) -> Point2<S> {
-        Point2::new(
-            a.x.partial_max(b.x),
-            a.y.partial_max(b.y)
-        )
+        Point2::new(a.x.partial_max(b.x), a.y.partial_max(b.y))
     }
 }
 
 impl<S> MinMax for Point3<S>
     where S: BaseNum
 {
     fn min(a: Point3<S>, b: Point3<S>) -> Point3<S> {
-        Point3::new(
-            a.x.partial_min(b.x),
-            a.y.partial_min(b.y),
-            a.z.partial_min(b.z)
-        )
+        Point3::new(a.x.partial_min(b.x),
```

Comment:
Do you have the latest version of `rustfmt`? I'm pretty sure it uses block format.